shared/middleware/helpers/property_bag.rb

```diff
@@ -114,6 +116,12 @@ def set(name, value, ip_address, time = DateTime.now)
       },
     )
     value
+  rescue Aws::DynamoDB::Errors::ValidationException => e
+    if e.message == 'Item size has exceeded the maximum allowed size'
+      too_large
```

Comment:
Here is one example of keeping response status codes out of helpers: https://github.com/code-dot-org/code-dot-org/blob/staging/shared/middleware/helpers/bucket_helper.rb#L30